lms/djangoapps/courseware/toggles.py

```diff
@@ -238,16 +251,6 @@ def streak_celebration_is_active(course_key):
     )
 
 
-# .. toggle_name: COURSES_INVITE_ONLY
-# .. toggle_implementation: SettingToggle
-# .. toggle_type: feature_flag
-# .. toggle_default: False
-# .. toggle_description: Setting this sets the default value of INVITE_ONLY across all courses in a given deployment
-# .. toggle_category: admin
-# .. toggle_use_cases: open_edx
-# .. toggle_creation_date: 2019-05-16
-# .. toggle_expiration_date: None
-# .. toggle_tickets: https://github.com/mitodl/edx-platform/issues/123
-# .. toggle_status: unsupported
-def is_courses_default_invite_only_enabled():
-    return SettingToggle("COURSES_INVITE_ONLY", default=False).is_enabled()
+def course_is_invitation_only(courselike) -> bool:
```

Comment:
Does "courselike" have some idiomatic meaning?

Reply:
Ah fair question!

We have two very similar objects in the platform - the actual mongo course block and a CourseOverview object (i.e. a row of the CourseOverview model).

They have a lot of fields in common (intentionally -- CourseOverview is the LMS-side SQL cache of some mongo course metadata info -- like learning sequences, but much older).

So sometimes code that just wants to check a field that exists on both will accept either type of object -- easily done in python. And often uses a variable like `courselike` to indicate that.